src/mongocxx/test/spec/operation.cpp

```diff
@@ -204,7 +204,7 @@ document::value operation_runner::_run_distinct(document::view operation) {
     }
 
     stdx::optional<cursor> result_cursor;
-    if (client_session* session = _lookup_session(operation["arguments"].get_document().value)) {
+    if (auto session = _lookup_session(operation["arguments"].get_document().value)) {
```

Comment:
Nice use of auto!  Even if this method wasn't changing auto is definitely better.